code/modules/mob/mob.dm

```diff
@@ -1213,7 +1213,7 @@ mob/proc/yank_out_object()
 		to_chat(host, "<span class='info'>You are now a mouse. Try to avoid interaction with players, and do not give hints away that you are more than a simple rodent.</span>")
 
 /mob/proc/assess_threat() //For sec bot threat assessment
-	return
+	return 5
```

Comment:
There was a change to secbot threat analysis.  Before it was impossible for anything that wasn't a carbon/human to have a natural threat value other than 5.

This was done so that true devils have a threat level of 666.